src/main/java/models/GameEngine.java

```diff
@@ -166,40 +168,46 @@ public boolean isCardKilled(Card card) {
     }
 
     public void attack(Card currentPlayerCard, Card opponentCard) {
+        boolean playerAttack = false;
         int currentPlayerAttack = currentPlayerCard.attack();
-        int opponentPlayerAttack = opponentCard.attack();
+        int opponentPlayerAttack;
+        try {
+            opponentPlayerAttack = opponentCard.attack();
+        } catch (NullPointerException e) {
+            System.out.println("Attacking Opponent!");
+            playerAttack = true;
+            opponentPlayerAttack = new Card().attack();
```

Comment:
Varför skapas ett nytt kort här? Blir en ny attack dmg och inte dmg från opponentCard som tas in som argument.

Reply:
Ja, som det var innan så tog player skada från currentplayers attack utan att kunna försvara sig (slå tärning) , antar att moståndaren ska slå tärningen oavsett om attacken är mot ett kort eller själva playern.  

Det blir bara ett nytt kort om det skickas in attack(CPcard, null); 
Så det blir inget nytt kort om man har skickat in ett opponentcard.